modules/custom/az_migration/migrations/migrate_plus.migration.az_node_event.yml

```diff
@@ -0,0 +1,188 @@
+id: az_node_event
+label: AZ Event Content
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_node
+  node_type: uaqs_event
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_event
+  migration_dependencies:
+    requireed:
+      - az_event_categories
+
+process:
+  type: 
+    plugin: default_value
+    default_value: az_event
+  langcode:
+    plugin: static_map
+    bypass: true
+    source: language
+    map:
+      und: en
+  
+  title:
+    plugin: sub_process
+    source: title_field
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_subheading:
+    plugin: sub_process
+    source: field_uaqs_summary_short
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_link:
+    plugin: sub_process
+    source: field_uaqs_link
+    process:
+      delta: delta
+      uri: url
+      title: title
```

Comment:
The default value for `url` on the source D7 `field_uaqs_link` field is the `[node:url]` token, which doesn't work in D9.  I think we may want to add something similar to what you did for Carousel to replace `[node:url]` values either with an empty string or `<none>`.

Reply:
@smitahantzios This comment has still not been addressed.